### PR TITLE
2by2 images

### PR DESCRIPTION
**Pull request by TheDoplarEffect**
Our component with 4 images per page and five buttons underneath.
Added pazaaz with a animation changing between pages (getting this to work was 90% of time spent).
Each Image is a Link to the relavant location (not yet part of Main)

### CODE REVIEW

**Comment by TheDoplarEffect** [comment]
I've left the Storybook component used to design the item, however, now that we have the LoaderFucntion it no longer works.  File can be deleted if needs be

**Comment by TheDoplarEffect** [comment]
The final project already works on vite server.  under /movies

File: src/routes/Movies.tsx [hunk 1]
```diff
@@ -1,32 +1,70 @@
...
-  const result = await axios.get<MoviesType[]>("http://localhost:3000/movies");
-  return result.data;
+  return API.fourByFour();
 }
-
 export function Movies() {
-  const movies = useLoaderData() as MoviesType[];
+  const API = useLoaderData() as Movie[][];
+  const [number, setNumber] = useState(0);
+  const [previousvalue, setpreviousvalue] = useState(0);
```
Comment (ReallyBoard123) [inline src/routes/Movies.tsx:15]: naming convention

File: src/components/MoviesPageButton.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import clsx from "clsx";
+import { useState } from "react";
+
+type Input = {
+  number: number;
```
Comment (ReallyBoard123) [inline src/components/MoviesPageButton.tsx:5]: please use a different binding name

File: src/components/MoviesPageButton.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  setNumber: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export default function Moviespagebutton({ number, setNumber }: Input) {
```
Comment (ReallyBoard123) [inline src/components/MoviesPageButton.tsx:9]: use PascalCase

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]
this is one of the best looking components in the project so far! and well done with the extra effort on the animation. Could you please go through all the files and use the proper naming conventions?

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]
previous comments still not resolved

File: src/components/MoviesPageButton.tsx (hunk 3 above)
Comment (peelerzin) [reply src/components/MoviesPageButton.tsx]: This one is now changed using PascalCase.  MoviesPageButton.

File: src/routes/Movies.tsx (hunk 1 above)
Comment (peelerzin) [reply src/routes/Movies.tsx:15]: solved.

File: src/components/MoviesPageButton.tsx (hunk 2 above)
Comment (peelerzin) [reply src/components/MoviesPageButton.tsx:4]: solved with other binding name

File: src/components/MoviesPageButton.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import clsx from "clsx";
+
+type InputProps = {
+  inputNumber: number;
+  setNumber: React.Dispatch<React.SetStateAction<number>>;
```
Comment (ReallyBoard123) [inline src/components/MoviesPageButton.tsx:5]: would it not be setInputNumber?

File: src/routes/Movies.tsx (hunk 1 above)
Comment (ReallyBoard123) [reply src/routes/Movies.tsx:15]: was it resolved?
if it was... why do we still have this in lowercase here:
const [previousvalue, setpreviousvalue]

**Comment by TheDoplarEffect** [comment]
Corrected typing...

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]
... @peelerzin, that was confusing to go through the file and then read a comment from you saying that something has been resolved when it was not. 

Your component looks well done! I am waiting forward to merging it! But please, please go through all the files and change the names according to the conventions. It is very frustrating to go through each line of code every single time of this pull request and point at the same thing again and again

**Comment by peelerzin** [comment]
We tried with William to fix it this morning, now we going to respect all the conventions by checking all the pages. :)

**Review by ReallyBoard123 — APPROVED** [review]
you did it!

**Pull request merged**